src/metabase/search/config.clj

```diff
@@ -39,7 +39,7 @@
 
```

Comment:
Note that according to my linter, this is dead code. IDK if we do some sort of dynamic magic with it, but I couldn't find any other references to it. Should we ✂️ ?

Reply:
@markbastian what is dead code? I don't see anything highlighted by this comment. Seems it's just orphaned in the UI
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/6377293/214972522-bdaa5af1-6794-4807-b5cc-5af0ae889542.png">

Reply:
@dpsutton he must be referring to `searchable-db-models` (L40 below). Indeed, it's not used any more. Looking at `git blame`, you initially wrote it for use in the `DBModel` schema in the API, and `DBModel` and its usage were removed by Cam when upgrading to Toucan 2:

```clojure
- (s/defn ^:private from-clause-for-model :- [(s/one [(s/one DBModel "model") (s/one s/Keyword "alias")]
+ (s/defn ^:private from-clause-for-model :- [(s/one [(s/one s/Keyword "table name") (s/one s/Keyword "alias")]
```

So yeah, makes sense that it's gone (since Toucan 2 doesn't use classes the way Toucan 1 does), makes sense that Cam missed that `searchable-db-models` is no longer used since it was a huge PR, and makes sense to remove it now. Good catch, Mark!